Hangman/Hangman/main.swift

```diff
@@ -2,11 +2,167 @@
+let hangManDrawing = ["""
+-----|
+|
+|
+|
+|
+|
+--------
+""",
+                      """
+-----|
+|    O
+|
+|
+|
+|
+--------
+""",
+                      """
+-----|
+|    O
+|    |
+|
+|
+|
+--------
+""",
+                      """
+-----|
+|    O
+|    |\\
+|
+|
+|
+---------
+""",
+                      """
+-----|
+|    O
+|   /|\\
+|
+|
+|
+---------
+""",
+                      """
+-----|
+|    O
+|   /|\\
+|     \\
+|
+|
+---------
+""",
+                      """
+-----|
+|    O
+|   /|\\
+|   / \\
+|
+|
+---------
+"""
+]
 
-print("Hello, World!")
+//var gameState = true
```

Comment:
Please remove commented out code before submitting, practice makes permanent so you want to start getting into these good habits early on :)